projects/common/src/utilities/formatters/string/string-formatter.ts

```diff
@@ -42,3 +42,10 @@ export const displayString = (provided?: unknown): string => {
 export const collapseWhitespace = (str: string): string =>
   // Replace all whitespace with a single space
   str.replace(/\s\s+/g, ' ');
+
+export const getStringsFromCommaSeparatedList = (text: string): string[] =>
+  text
+    .trim()
```

Comment:
nit: this is redundant, would function the same without since we're trimming the components

Reply:
done